bounswe/bounswe2017group5
# Semantic tagging UI

## About

Connect our `search_wiki` endpoint to the frontend and let our users add semantic tags to their groups on group creation page.

### Changes

- Add `select2` dependency for dynamic (and cool 😎 ) selectbox fields.
- Implement the backend connection of semantic tags and Tag objects.
- Implement the frontend for dynamic tag selection.

## Visuals

![test-semtag](https://user-images.githubusercontent.com/13895224/33342411-0f49c978-d493-11e7-906c-c1ce4c78e06d.gif)

## Testing

- Go to the group creation page.
- Add some tags by typing in a label in the tag field and selecting the appropriate tags.
- Create the group.
- Hover on the tag names to see the descriptions on the group details page.
